src/target/parsers/cpu.cc

```diff
@@ -30,10 +30,10 @@ namespace cpu {
 
 Optional<String> DetectSystemTriple() {
   auto pf = tvm::runtime::Registry::Get("target.llvm_get_system_triple");
-  if (pf->defined()) {
-    return (*pf)();
-  }
-  return {};
+  ICHECK(pf != nullptr)
```

Comment:
Apologies for the confusion, we'd still need to wrap this section in `#ifdef TVM_LLVM_VERSION` like before

Reply:
Sorry for the inconvenience, I got a bit confused, but i think i fixed it now.

Reply:
No worries :)